frodo.js

```diff
@@ -1,27 +1,27 @@
-const oddNumbers = numbers.filter(num => num % 2 !== 0);
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.